gravitee-rest-api-management/gravitee-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiResource.java

```diff
@@ -318,9 +318,11 @@ public Response deleteApi() {
     @Permissions({
             @Permission(value = RolePermission.API_DEFINITION, acls = RolePermissionAction.UPDATE)
     })
-    public Response deployApi() {
+    public Response deployApi(
+            @ApiParam(name = "apiDeployment", required = true)
```

Comment:
Should not be required because adding a mandatory parameter is a breaking change.

Reply:
resolved by 4403464